hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.tsx

```diff
@@ -3,14 +3,36 @@
  */
 
 import React from 'react';
-import {PropTypes} from 'prop-types';
+import PropTypes from 'prop-types';
 import Form from 'react-bootstrap/Form';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Button from 'react-bootstrap/Button';
 
 export const classificationTypeTBD = 'TBD';
 
+type Action = {
```

Comment:
This is a larger concern probably beyond the scope of this PR but I would strongly prefer things like `Action` and `Classification` actually correspond to something on the backend in python. 

Right now these models makes sense for the UI but don't map back as well. The more the UI interface and the actually backing structure diverge the more difficult to develop on and imo less intuitive the system will become.

Reply:
At a minimum can you add some docstrings to these types?

Reply:
This doesnt map 1:1 with anything in python. Its effectively a pointer an Action. The Action Settings tab would have the 1:1 mapping with `WebhookActionPerformer` in python